rustls/src/client/builder.rs

```diff
@@ -8,7 +8,7 @@ use crate::builder::{ConfigBuilder, WantsVerifier};
 use crate::client::{handy, ClientConfig, EchMode, ResolvesClientCert};
```

Comment:
I think this commit title has a typo: "SingleCertAndReady" -> "SingleCertAndKey"

Reply:
Nice catch! Brains...